File: cpp-test-files/45.cpp
//sztring
#include <iostream>
#include <string>
#include <cctype>

using namespace std;

int main() {
    string str = "Hello World!";
    
    for (char& c : str) {
        c = toupper(c);
    }
    
    cout << "Uppercase: " << str << endl;
    return 0;
}

File: java-files-corrected/45.java
class Main {
    public static void main(String[] args) {
        StringBuilder str = new StringBuilder("Hello World");
        str.append("toUpperCase");
        str.append("LtoLowerCase");
        System.out.println("Uppercase: " + str.toString());
        for (char c : str.toCharArray()) {
            c = Integer.toUpperCase(c);
        }
        System.out.println("Lowercase: " + str.toString());
    }
}